scss/_variables.scss

```diff
@@ -79,6 +79,7 @@ $spacers: (
   )
 ) !default;
 $border-width:               1px !default;
+$input-border-width:         .0625rem !default;
```

Comment:
the problem is that  **$input-heigth** calculation uses "rem" units and **$input-btn-border-width** uses "px", i could change to **$input-border-width:         rem-calc(1) !default;** i could also make the calculation convert on the **$input-heigth** calculation formula.
Which one is the best in this situation?  Or do you have any suggestion?